src/stream/mod.rs

```diff
@@ -133,4 +146,18 @@ pub trait Stream: Send + 'static {
     {
         flat_map::new(self)
     }
+
+    fn skip_while<P>(self, pred: P) -> SkipWhile<Self, P>
+        where P: FnMut(&Self::Item) -> Result<bool, Self::Error> + Send + 'static,
```

Comment:
Oh I might actually expect this to return `bool` rather than `Result<bool, Self::Error>`